cmd/trace-agent/config/remote/config.go

```diff
@@ -74,7 +76,15 @@ func ConfigHandler(r *api.HTTPReceiver, client remote.ConfigUpdater, cfg *config
 		}
 		cfg, err := client.ClientGetConfigs(req.Context(), &configsRequest)
 		if err != nil {
-			statusCode = http.StatusInternalServerError
+			if e, ok := status.FromError(err); ok {
+				switch e.Code() {
+				case codes.Unimplemented:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏So we return OK when `Unimplemented`?

Reply:
I would return 404, as the feature is not really there